src/components/SaveTaskForm.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+
+const request = require('superagent');
```

Comment:
@ajvb Let's not use superagent, which caused a bunch of problems for me. Let's use [fetch](https://github.com/github/fetch), which I've already added as a dependency in `index.html`.